src/pages/lib/graphqlapi/relational/q/platform/[platform].mdx

```diff
@@ -0,0 +1,835 @@
+    Log.e("MyAmplifyApp", "Failed to fetch post", error)
+}
+```
+
+This will populate the selection set of the post in the GraphQL document which indicates to your GraphQL service to retrieve the post model as part of the operation. Once the comment is loaded, the post model is immediately available in-memory without requiring an additional network request.
+
+</Block>
+</BlockSwitcher>
+
+</InlineFilter>
+
+Query for the `Post` and the first page of comments for the post: 
+
+<InlineFilter filters={["android"]}>
+
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+Amplify.API.query(
+    ModelQuery.<Post, PostPath>get(
+        Post.class,
+        new Post.PostIdentifier("p1"),
+        (postPath -> includes(postPath.getComments()))
+    ),
+    response -> {
+        Post post = response.getData();
+        ModelList<Comment> commentsModelList = post.getComments();
+        if (commentsModelList instanceof LoadedModelList) {
+            List<Comment> comments = ((LoadedModelList<Comment>) commentsModelList).getItems();
+            Log.i("MyAmplifyApp", "Comments: " + comments);
+        }
+    },
+    failure -> Log.e("MyAmplifyApp", "Failed to fetch post", failure)
+);
+```
+
+The network request for post includes the comments, eagerly loading the first page of comments in a single network call.
+
+</Block>
+<Block name="Kotlin - Callbacks">
+
+```kotlin
+Amplify.API.query(
+    ModelQuery.get<Post, PostPath>(
+        Post::class.java,
+        Post.PostIdentifier("p1")
+    ) { postPath ->
+        includes(postPath.comments)
+    },
+    { response ->
+        val post = response.data
+        val comments = (post.comments as? LoadedModelList)?.items
+        Log.i("MyAmplifyApp", "Comments: $comments")
+    },
+    { Log.e("MyAmplifyApp", "Failed to fetch post", it) }
+)
+```
+
+The network request for post includes the comments, eagerly loading the first page of comments in a single network call.
+
+</Block>
+<Block name="Kotlin - Coroutines">
+
+```kotlin
+try {
+    val post = Amplify.API.query(
+        ModelQuery.get<Post, PostPath>(
+            Post::class.java,
+            Post.PostIdentifier("p1")
+        ) { postPath ->
+            includes(postPath.comments)
+        }
+    ).data
+    val comments = (post.comments as? LoadedModelList)?.items
+    Log.i("MyAmplifyApp", "Comments: $comments")
+} catch (error: ApiException) {
+    Log.e("MyAmplifyApp", "Failed to fetch post", error)
+}
+```
+
+The network request for post includes the comments, eagerly loading the first page of comments in a single network call.
+
+</Block>
+</BlockSwitcher>
+
+You can generate complex nested queries through the includes parameter.
+
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+ModelQuery.get<Comment, CommentPath>(Comment::class.java, "c1") { commentPath ->
+    includes(commentPath.post.comments)
+}
+```
+The network request for post includes the comments, eagerly loading the first page of comments in a single network call.
```

Comment:
Should this be - 
The network request for comments includes the post, eagerly loading the first page of comments in a single network call?

Reply:
Good catch. Corrected.